### DISCUSSION TITLE
Is there a way to debug draw ray/shape casts?

### DISCUSSION THREAD

**Discussion by jfaz1** [jrouwe/JoltPhysics#1419] [Q&A] [answered]
Hello, the built-in debug renderer works fine for bodies, but doesn't seem like its meant to be able to draw ray/shape casts. Is there a recommended way to debug draw casts?

**Comment by jrouwe** [comment, marked as answer]
You would have to call `DebugRenderer::DrawLine` yourself whenever you do a ray cast, there's no automated way to draw all ray casts.

**Reply by jfaz1** [reply]
Gotcha, that's basically what I'm doing already; thanks!
